merkletree/str.go

```diff
@@ -21,10 +21,10 @@ type SignedTreeRoot struct {
 	PreviousEpoch   uint64
 	PreviousSTRHash []byte
 	Signature       []byte
-	Policies        Policies
+	Policies        *Policies
```

Comment:
Isn't the overhead massively dominated by the cloned trees in memory?  The tradeoff is the potential for a corrupted state, which seems worth it.

Unfortunately, documenting that sort of expectation is unreliable and I would caution to err on the side of programming defensively.